packages/wonder-blocks-icon-button/components/icon-button.test.js

```diff
@@ -30,4 +37,61 @@ describe("IconButton", () => {
             ),
         ).toThrowError("Light is only supported for primary IconButtons");
```

Comment:
I wish this were using Arrange, Act, Assert. Having it all in an expect might be terse, but it's horrid.